### PR TITLE
Updates SystemFactory class to numpydoc Docstring Format

### PR DESCRIPTION
**Pull request by megosato**
In this PR, I have reformatted the docstrings for the SystemFactory class to follow the numpydoc docstring formatting guidelines. This addressed issue #24

### CODE REVIEW

**Comment by megosato** [comment]
@nathanmlim I think I updated the PR to the finished docstrings, but please let me know if it didn't work, I'm a little confused

File: blues/simulation.py [hunk 1]
```diff
@@ -21,31 +23,39 @@
...
     """
-    SystemFactory contains methods to generate/modify the OpenMM System object required for
-    generating the openmm.Simulation using a given parmed.Structure()
+    SystemFactory contains methods to generate/modify the OpenMM System object
+    required for generating the openmm.Simulation using a given
+    parmed.Structure()
 
-    Usage Example
-    -------------
+    Example
```
Comment (nathanmlim) [inline blues/simulation.py]: This should be `Examples`

File: blues/simulation.py [hunk 2]
```diff
@@ -21,31 +23,39 @@
...
+    -------
     #Load Parmed Structure
```
Comment (nathanmlim) [inline blues/simulation.py]: For formatting examples, it looks better if you do a short blurb and then do the code block:

Format the below to something like:

```
Load Parmed Structure from our input files, select the move type, initialize the MoveEngine, and generate the openmm systems.

>>> structure = parmed.load_file('eqToluene.prmtop', xyz='eqToluene.inpcrd')
>>> ligand = RandomLigandRotationMove(structure, 'LIG')
>>> ligand_mover = MoveEngine(ligand)
>>> systems = SystemFactory(structure, ligand.atom_indices, config['system'])
```

Note above that spacing matters. The comments/blurb doesn't need the `#` but there needs to be a space between the sentence and the code block itself.

File: blues/simulation.py [hunk 3]
```diff
@@ -21,31 +23,39 @@
...
+    >>> systems.alch
 
     #Freeze atoms in the alchemical system
-    systems.alch = SystemFactory.freeze_atoms(systems.alch,
-                                            freeze_distance=5.0,
-                                            freeze_center='LIG'
-                                            freeze_solvent='HOH,NA,CL')
+
+    >>> systems.alch = SystemFactory.freeze_atoms(systems.alch,
+    ...                                     freeze_distance=5.0,
```
Comment (nathanmlim) [inline blues/simulation.py]: You don't need the `...` here or the next few lines.

File: blues/simulation.py [hunk 4]
```diff
@@ -398,12 +443,19 @@ def freeze_radius(cls,
...
-            AmberMask selection in which to select solvent atoms for zeroing their masses.
+            AmberMask selection in which to select solvent atoms for zeroing
+            their masses.
+
+        Returns
+        -------
+        system : openmm.System
+            Modified system with masses outside the `freeze center` zeroed.
 
         References
```
Comment (nathanmlim) [inline blues/simulation.py]: Can you format references like below:

```
    Notes
    -----
    .. [1] I. Andricioaei, J. E. Straub, and A. F. Voter, J. Chem. Phys. 114, 6994 (2001).
        https://doi.org/10.1063/1.1358861
```

File: blues/simulation.py [hunk 5]
```diff
@@ -398,12 +443,19 @@ def freeze_radius(cls,
         Kwargs
```
Comment (nathanmlim) [inline blues/simulation.py]: Can you move `Kwargs` sections into the `Parameters` sections?  The `Kwargs` section is just an alias for `Parameters` so you can just combine them into a `Parameters` section.

File: blues/simulation.py [hunk 6]
```diff
@@ -4,6 +4,8 @@
 Authors: Samuel C. Gill
 Contributors: Nathan M. Lim, David L. Mobley
```
Comment (nathanmlim) [inline blues/simulation.py]: You can add your name after mine and before David's here :+1:

**Comment by nathanmlim** [comment]
Merged.

**Pull request merged**